src/main/java/ch/hevs/cloudio/endpoint/CloudioFactoryObject.java

```diff
@@ -0,0 +1,12 @@
+package ch.hevs.cloudio.endpoint;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Using
```
@JsonAnySetter
private Map<String, Object> properties = new LinkedHashMap<>();
```
you can automate deserialisation of the additional properties to a map.

Reply:
I will add those properties map 
Also I was thinking to remove those Json annotation to not make those classes depended of Jackson and configure the object Mapper to ignore the unused fields. What do you think?